RepositoryCompiler/CodeModel/CodeParsers/CSharp/CSharpCodeParser.cs

```diff
@@ -105,7 +106,8 @@ private List<CaDETField> ParseFields(IEnumerable<MemberDeclarationSyntax> nodeMe
                     {
                         Name = field.Identifier.Text,
                         Parent = parent,
-                        Modifiers = GetModifiers(node)
+                        Modifiers = GetModifiers(node),
+                        Type = new CaDETClass() { Name = fieldDeclaration.Declaration.Type.ToString()}
```

Comment:
I used the Name attribute (not FullName), because when declaring fields, type is just _name_ of the type (not _namespace.name_).

Reply:
So long as you are sure this will uniquely identify the type. Did you consider situations where there are multiple classes with the same name (but in a different namespace)?

Reply:
Changed - full name is taken into account.